src/core/InternalAnimatedValue.js

```diff
@@ -31,6 +31,8 @@ export default class InternalAnimatedValue extends AnimatedNode {
     );
   }
 
+  _offset = 0;
```

Comment:
I don't think this is needed. Looks like the `_offset` param's been left here by mistake. We never assign to it, so maybe we can clean it up from `__onEvaluate` completely?

Reply:
This needs to be here to make the `imperative` demo work. This is set when you call `setOffset`. It must be 0 for all other cases or you get NaN